cmake/linux/package_linux.sh.in

```diff
@@ -56,28 +58,27 @@ export PATH
 PATH="$(dirname "@QT_QMAKE_EXECUTABLE@")":$PATH
 
 # Fetch portable linuxdeployqt if cache is older than $DAYSOLD
+APPIMAGE=$USERBIN/linuxdeployqt.AppImage
 echo -e "\nDownloading linuxdeployqt to ${LINUXDEPLOYQT}..."
-mkdir -p "$HOME/bin"
+mkdir -p "$USERBIN"
 DAYSOLD=2
 if env -i which linuxdeployqt > /dev/null 2>&1; then
 	skipped "System already provides this utility"
-elif ! find "$LINUXDEPLOYQT" -mtime -$DAYSOLD 2>/dev/null|grep -q "." > /dev/null 2>&1; then
+elif ! find "$APPIMAGE" -mtime -$DAYSOLD 2>/dev/null|grep -q "." > /dev/null 2>&1; then
```

Comment:
Just a side-note, I wrote-but-passionately-hate having this internal logic.  If there's a submodule we can clone offering a `curl|wget` binary caching method, we could simplify this.